frontend/www/js/omegaup/components/arena/ContestListv2.vue

```diff
@@ -12,31 +12,85 @@
         vertical
         nav-wrapper-class="contest-list-nav col-sm-4 col-md-2"
       >
-        <div class="card-body">
-          <div class="row">
-            <div class="col-md-6">
-              <form :action="queryURL" method="GET">
-                <div class="input-group">
-                  <input
-                    v-model="currentQuery"
-                    class="form-control"
-                    type="text"
-                    name="query"
-                    autocomplete="off"
-                    :placeholder="T.wordsKeyword"
-                  />
-                  <div class="input-group-append">
+        <b-card>
+          <b-container>
+            <b-row class="p-1" align-v="center">
+              <b-col cols="6">
+                <form :action="queryURL" method="GET">
+                  <div class="input-group">
                     <input
-                      class="btn btn-primary btn-md active"
-                      type="submit"
-                      :value="T.wordsSearch"
+                      v-model="currentQuery"
+                      class="form-control"
+                      type="text"
+                      name="query"
+                      autocomplete="off"
+                      :placeholder="T.wordsKeyword"
                     />
+                    <div class="input-group-append">
+                      <input
+                        class="btn btn-primary btn-md active"
+                        type="submit"
+                        :value="T.wordsSearch"
+                      />
+                    </div>
                   </div>
-                </div>
-              </form>
-            </div>
-          </div>
-        </div>
+                </form>
+              </b-col>
+              <b-col cols="6">
+                <b-dropdown ref="dropdownOrderBy" no-caret>
+                  <template #button-content>
+                    <div>
+                      <font-awesome-icon icon="sort-amount-down" />
+                      {{ T.contestOrderBy }}
+                    </div>
+                  </template>
+                  <b-dropdown-item href="#" @click="orderByName">
```

Comment:
Acá para poder acceder a este elemento en las pruebas, le pudieras agregar un [`data-attribute`](https://developer.mozilla.org/en-US/docs/Learn/HTML/Howto/Use_data_attributes):
```suggestion
                  <b-dropdown-item
                    href="#"
                    @click="currentOrder = OrderBy.Name"
                    data-order-by-name
                  >
```
?

Lo mismo para todos los demás items